### ISSUE TITLE
Is it possible to access the  Spock.lang.Title annotation value of each spec from summary report

### ISSUE THREAD

**Issue by SueNag** [renatoathaydes/spock-reports#146]
I want to create a single page report with all the details for the business and using class names for the spec stats doesn't make mush sense to them. I see that TemplateReportAggregator is filtering selected info from SpecData. Is there a way to extend it so that I get all the Spec info in the summary Report?

**Comment by renatoathaydes** [comment]
The data seems to be all available in the summary report.

See [this line](https://github.com/renatoathaydes/spock-reports/blob/master/src/main/groovy/com/athaydes/spockframework/report/template/TemplateReportAggregator.groovy#L41):

```groovy
    engine.createTemplate( template )
            .make( [ data          : allData,
                     'utils'       : Utils,
                     'fmt'         : new StringFormatHelper(),
                     projectName   : projectName,
                     projectVersion: projectVersion ] )
    .toString()
```

`data` seems to contain everything... can you specify something that you think is missing from it?

**Comment by renatoathaydes** [comment]
That's a good idea, it is not hard to add further information to the `data` field, but notice that this data is serialized as JSON so it needs to be just simple data like spec title, narrative etc. If there's any specific information about the spec you want, let me know.

**Comment by SueNag** [comment]
I was thinking more along the lines of adding a map of specData (spec name to Spec Data) like spec stats. The requirement was to generate one long report of summary file & all spec files like what Cucumber does . I do see your point about json serialization.

**Comment by SueNag** [comment]
I think having spec title and narrative would make the Summary report much more readable.

**Comment by renatoathaydes** [comment]
Released version 1.6.0 which includes in the `data` Map the following additional fields: `title` and `narrative`.

I also added `title` to the HTML summary report, by default, and allowed that to be configured so you can show class name, title, or both.

**Comment by SueNag** [comment]
Awsome, I will pull in the latest version. Thanks a bunch.
